client/state/timezones/test/reducer.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { useSandbox } from 'test/helpers/use-sinon';
+
+// Reducers
+import timezonesReducer, {
+	items as itemsReducer,
+	requesting as requestReducer
+} from '../reducer';
+
+/**
+ * Actions creators functions
+ */
+import {
+	timezonesReceiveAction,
+	timezonesRequestSuccessAction,
+	timezonesRequestFailureAction,
+	timezonesRequestAction,
+} from '../actions';
+
+/**
+ * Fixture data
+ */
+import { TIMEZONES_SYNC_DATA } from './fixture';
+
+describe( 'reducer', () => {
+	let sandbox;
+
+	useSandbox( newSandbox => {
+		sandbox = newSandbox;
+		sandbox.stub( console, 'warn' );
+	} );
+
+	it( 'should export expected reducer keys', () => {
+		expect( timezonesReducer( undefined, {} ) ).to.have.keys( [
+			'items',
+			'requesting'
+		] );
+	} );
+
+	describe( '#items()', () => {
+		it( 'should default to an empty Array', () => {
+			expect( itemsReducer( undefined, {} ) ).to.eql( {} );
+		} );
+
+		it( 'should index items state', () => {
+			const initialState = undefined;
+			const timezones = TIMEZONES_SYNC_DATA;
+			const action = timezonesReceiveAction( timezones );
+
+			const expectedState = timezones;
+			deepFreeze( expectedState );
```

Comment:
We generally only need to deep freeze initialState unless you think the tests might manipulate the fixture data. If you think that might be the case for the fixture, maybe consider moving that test setup out of these `it` blocks.